sms/src/main/java/org/motechproject/sms/http/MultilineSingleResponseHandler.java

```diff
@@ -50,7 +49,7 @@ public void handle(OutgoingSms sms, String response, Header[] headers) {
             getLogger().info(String.format("Sent messageId %s '%s' to %s", messageId, messageForLog(sms),
                     sms.getRecipients().get(0)));
             getAuditRecords().add(new SmsRecord(getConfig().getName(), OUTBOUND, sms.getRecipients().get(0),
-                    sms.getMessage(), now(), DeliveryStatus.DISPATCHED, null, sms.getMotechId(), messageId, null));
+                    sms.getMessage(), now(), getTemplateOutgoingResponse().getSuccessStatus(), null, sms.getMotechId(), messageId, null));
```

Comment:
@sebbrudzinski
MOTECH member
sebbrudzinski added a note 30 seconds ago

According to the javadocs, this field contains the status, that the provider considers as success. Are you sure this is the proper field to use? Won't this always put the success in the logs, no matter what the actual response was?